### PR TITLE
i18n(fr): Updating `framework-components` translation

### PR DESCRIPTION
**Pull request by Chrissdroid**
#### What kind of changes does this PR include?

- [x] New or updated content
- [x] Translated content

#### Description

- What does this PR change? Updates content from `src/pages/fr/core-concepts/framework-components.md` to reflect latest changes

### CODE REVIEW

File: src/pages/fr/core-concepts/framework-components.md [hunk 1]
```diff
@@ -112,29 +114,98 @@ import MyVueComponent from '../components/MyVueComponent.vue';
...
+// src/pages/MyAstroPage.astro
+---
+import MyReactSidebar from '../components/MyReactSidebar.jsx';
+---
+<MyReactSidebar>
+  <p>Voilà un panneau latéral avec du texte et un bouton.</p>
+</MyReactSidebar>
+```
+
+De plus, vous pouvez les utiliser ["Slots" Nommés](/fr/core-concepts/astro-components/#emplacements-nommés) pour grouper des enfants spécifiques ensemble.
```
Comment (delucis) [inline src/pages/fr/core-concepts/framework-components.md:136]: Typo on word order?

```suggestion
De plus, vous pouvez utiliser les ["Slots" Nommés](/fr/core-concepts/astro-components/#emplacements-nommés) pour grouper des enfants spécifiques ensemble.
```

File: src/pages/fr/core-concepts/framework-components.md [hunk 2]
```diff
@@ -112,29 +114,98 @@ import MyVueComponent from '../components/MyVueComponent.vue';
...
+:::
 
-Chaque Framework a son propre modèle d'imbrication : par exemple, les propriétés `children` et [`render`](https://reactjs.org/docs/render-props.html) pour React et Solid. Ou bien même des balises `<slot />` avec ou sans noms pour Svelte et Vue.
+Ceci permet de construire des applications entières dans votre Framework JavaScript préféré et de les rendre via un composant parent, à une page Astro.
 
-Notez cependant que vous ne pouvez pas passer des propriétés `render` ou des `<slot />` nommés à un composant de Framework depuis un fichier `.astro`, même si le composant de Framework le supporte. Cela est dû à une limitation du compilateur de Astro.
+:::note
+Les composants sont toujours rendus en HTML statique, même lorsqu'ils contiennent des composants de Framework qui sont hydratés. Cela signifie que vous ne pouvez pas passer des propriétés `render` à un composant de Framework depuis un composant Astro, cela ne fonctionnera pas car les composants Astro ne peuvent pas fournir le comportement client requis par ce modèle. À la place, vous pouvez utiliser les "Slots" nommés.
```
Comment (delucis) [inline src/pages/fr/core-concepts/framework-components.md:207]: Does this read OK? Two “cela” made for a very long sentence.

```suggestion
Les composants sont toujours rendus en HTML statique, même lorsqu'ils contiennent des composants de Framework qui sont hydratés. Cela signifie que vous ne pouvez pas passer des propriétés `render` à un composant de Framework depuis un composant Astro. Les composants Astro ne peuvent pas fournir le comportement client requis par ce modèle. À la place, vous pouvez utiliser les "Slots" nommés.
```

**Review by delucis — COMMENTED** [review]
Thanks for these updates @Chrissdroid! Made a couple of suggestions — let me know what you think. 💜

**Comment by Chrissdroid** [comment]
TYSM @delucis for taking your time on this ! All the suggestions make sense, now some sentence make actually sense 😆

**Comment by Yan-Thomas** [comment]
Nice work @Chrissdroid! There is a small change from #862 that I'm not sure if it impacts your translation, just pointing this out if you want to add it as part of this PR. Let me know when this is good to merge! 🚀

**Comment by Chrissdroid** [comment]
> Nice work @Chrissdroid! There is a small change from #862 that I'm not sure if it impacts your translation, just pointing this out if you want to add it as part of this PR. Let me know when this is good to merge! 🚀

Sorry for the delay on this @Yan-Thomas ! Thanks for notifying me on this ! 💯

**Comment by Yan-Thomas** [comment]
LGTM. Thanks @Chrissdroid!

**Pull request merged**